indico/modules/rb/client/js/components/TimeInformation.jsx

```diff
@@ -79,7 +79,12 @@ function TimeInformation({
             <div>
               <div>
                 <Icon name="sync alternate" />
-                <strong>{renderRecurrenceWeekdays(recurrenceWeekdays)}</strong>
+                <strong>
+                  <Translate>
+                    Every{' '}
+                    <Param name="weekdays" value={renderRecurrenceWeekdays(recurrenceWeekdays)} />
+                  </Translate>
+                </strong>
```

Comment:
You can also do this (but either way is fine):
```suggestion
                <Translate as="strong">
                  Every{' '}
                  <Param name="weekdays" value={renderRecurrenceWeekdays(recurrenceWeekdays)} />
                </Translate>
```

Reply:
Added